packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -182,6 +182,14 @@ def lb_enabled():
     return test_helpers.expanded_config['enable_lb'] == 'true'
 
 
+def bridge_nf_call_iptables():
```

Comment:
Should we say check_bridge_nf_call_iptables()? It is not clear what this function is supposed to do by its name